docs/conf.py

```diff
@@ -15,17 +15,18 @@
 # Incase the project was not installed
 import os
 import sys
+sys.path.append(os.path.abspath('../..'))
```

Comment:
I think this takes you outside your repository (i.e. 1 level up)? Should it be here?

Reply:
Yes, that was a mistake. Thanks.

Reply:
I think `../..` is still 1 level up, as `..` would be the root level directory.

Reply:
Ugh, I had deleted the wrong one. Good catch, thanks :)